### ISSUE TITLE
File pile.rb disappeared from lib/grok/pure 

### ISSUE THREAD

**Issue by rrepen** [jordansissel/ruby-grok#17]
In the latest version of jls-grok-0.11.0 the lib/grok/pure/pile.rb is not present.
the previous one I use jls-grok-0.10.12 is OK.
 
Is this expected?

Thanks
Regards

**Comment by wiibaa** [comment]
I suppose it is indeed expected from this commit https://github.com/jordansissel/ruby-grok/commit/c9fab1234ce90607c92273346f1bd2db8fe202bd 
have a look to the sample.rb in the commit to see what to change.

**Comment by suyograo** [comment]
Yes this is expected. It was just a list abstraction on top of grok[s] and did not provide any other benefit. Any reason why you need it?
